### PR TITLE
Review call_long_callback reference documentation.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Part of #442.
Slightly modify the current Reference documentation.

### CODE REVIEW

File: src/taipy/gui/gui_actions.py [hunk 1]
```diff
@@ -205,20 +205,28 @@ def invoke_long_callback(
...
-        user_status_function (Optional(Callable[[State^, bool, ...], None])): The optional user-defined status function that is invoked at the end (of and possibly during) the runtime of the user_function.<br/>
-            The first parameter of this function **must** be a `State^`. The second parameter of this function **must** be a Union[`bool`, `int`] that will indicate a progression (`int`) or the final status (`bool`)
-        user_status_function_args (Optional[List|Tuple]): The remaining arguments ot the user status function, as a List.
-        period (int): The interval in milli-seconds at which the user_status_function is called (default value is 0).</br>
-            When set to a value < 500, the user_status_function is called only when the user_function is terminated.
+        state (State^): The `State^` instance, as received in any callback.
+        user_function (Callable[[...], None]): The function that will be run independently of Taipy GUI. Note
+          that this function must not use *state*, which is not persisted across threads.
+        user_function_args (Optional[List|Tuple]): The remaining arguments.
```
Comment (FredLL-Avaiga) [inline src/taipy/gui/gui_actions.py]: not remaining: The arguments to user_function

File: src/taipy/gui/gui_actions.py [hunk 2]
```diff
@@ -205,20 +205,28 @@ def invoke_long_callback(
...
+        user_status_function (Optional(Callable[[State^, bool, ...], None])): The optional user-defined status
+            function that is invoked at the end of and possibly during the runtime of *user_function*.<br/>
+            The first parameter of this function **must** be a `State^`. The second parameter of this function
+            **must** be a Union[`bool`, `int`] that will indicate whether a progression (`int`) or the final
+            status (`bool`).
+        user_status_function_args (Optional[List|Tuple]): The remaining arguments of the user status function.
+        period (int): The interval, in milliseconds, at which *user_status_function* is called.<br/>
+            The default value is 0, meaning no call to *user_status_function* will happen.</br>
```
Comment (FredLL-Avaiga) [inline src/taipy/gui/gui_actions.py]: 0 is the same as < 500: user_status_function will be called only at the end

**Review by FredLL-Avaiga — APPROVED** [review]
A couple of comments

File: src/taipy/gui/gui_actions.py (hunk 1 above)
Comment (FabienLelaquais) [reply src/taipy/gui/gui_actions.py]: 👍

File: src/taipy/gui/gui_actions.py (hunk 2 above)
Comment (FredLL-Avaiga) [reply src/taipy/gui/gui_actions.py]: if you don't want calls, set user_status_function to None

Comment (FabienLelaquais) [reply src/taipy/gui/gui_actions.py]: Right. Let me clarify.

File: src/taipy/gui/gui_actions.py [hunk 3]
```diff
@@ -220,16 +220,18 @@ def invoke_long_callback(
...
             function that is invoked at the end of and possibly during the runtime of *user_function*:
 
-            - The first parameter of this function **must** be a `State^`.
-            - The second parameter of this function **must** be a bool or an int.<br/>
+            - The first parameter of this function will be set to a `State^` instance.
+            - The second parameter of this function will be set to a bool or an int, depending on the
+              conditions under which it is called:
 
-               - If the status function receives a Boolean value, then:
+               - If the status function receives a bool value in its second parameter, then:
```
Comment (FredLL-Avaiga) [inline src/taipy/gui/gui_actions.py]: We're already in the second parameter context here ;-)

Comment (FredLL-Avaiga) [reply src/taipy/gui/gui_actions.py]: replace "If the status function receives ..."
by
"If this parameter is set to ..." ?

**Review by FredLL-Avaiga — APPROVED** [review]

File: src/taipy/gui/gui_actions.py (hunk 3 above)
Comment (FabienLelaquais) [reply src/taipy/gui/gui_actions.py]: Makes sense, unfortunately.

**Review by FredLL-Avaiga — APPROVED** [review]
Awesome

**Pull request merged**